app/controllers/favorites_controller.rb

```diff
@@ -13,10 +15,15 @@ def update
   end
 
   def destroy
-    pet = Pet.find(params[:pet_id])
-    favorite.remove_pet(pet.id)
+    if params[:pet_id].nil?
+      favorite.remove_all_pets
+      flash[:notice] = "Removed all pets from favorites"
+    else
+      pet = Pet.find(params[:pet_id])
+      favorite.remove_pet(pet.id)
+      flash[:notice] = "#{pet.name} has been removed from favorites"
+    end
     session[:favorites] = favorite.contents
```

Comment:
I'm not sure what this line is doing/ do we need it here?

Reply:
I think it was a copy / paste oops